src/content/docs/en/reference/modules/astro-config.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Config imports API Reference
+sidebar:
+  label: 'astro:config'
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 6
+---
+
+import Since from '~/components/Since.astro';
+
+<p><Since v="5.7.0" /></p>
+
+This virtual module `astro:config` exposes a non-exhaustive, serializable, type-safe version of the Astro configuration. There are two submodules for accessing different subsets of your configuration values: [`/client`](#imports-from-astroconfigclient) and [`/server`](#imports-from-astroconfigserver).
+
+All available config values can be accessed from `astro:config/server`. However, for code executed on the client, only those values exposed by `astro:config/client` will be available.  This protects your information by only making some data available to the client.
+
+## Imports from `astro:config/client`
+
+```js
+import {
+  i18n, 
+  trailingSlash,
+  base,
+  build,
+  site,
+} from "astro:config/client"
+```
+
+Use this submodule for client side code that is executed on the client.
```

Comment:
Additionally, it's a bit redundant to say "client-side code that is executed on the client" - that's the whole point! I guess it's safe to omit that last part and make it read the same as the server-side example paragraph below.

```suggestion
Use this submodule for client-side code:
```